go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/kubescape/backend v0.0.20
 	github.com/kubescape/go-logger v0.0.22
 	github.com/kubescape/k8s-interface v0.0.170
-	github.com/kubescape/storage v0.0.111
+	github.com/kubescape/storage v0.0.112-0.20240912075751-9a554eb26977
```

Comment:
use the release pls

Reply:
I need to merge https://github.com/kubescape/storage/pull/148
before